vm_dump.c

```diff
@@ -958,6 +958,28 @@ rb_dump_machine_register(const ucontext_t *ctx)
         dump_machine_register(mctx->__gregs[REG_S2+7], "s9");
         dump_machine_register(mctx->__gregs[REG_S2+8], "s10");
         dump_machine_register(mctx->__gregs[REG_S2+9], "s11");
+#   elif defined __loongarch64
+        dump_machine_register(mctx->__gregs[LARCH_REG_SP], "sp");
+        dump_machine_register(mctx->__gregs[LARCH_REG_S0], "s0");
+        dump_machine_register(mctx->__gregs[LARCH_REG_S1], "s1");
```

Comment:
`s0` and `s1` dump here first.

Reply:
Sorry, this patch has merged, should I resubmit a new pull request to fix this issue?

Reply:
> Sorry, this patch has merged, should I resubmit a new pull request to fix this issue?

Hello, this issue has been fixed in #7372.